mm2src/lp_swap/maker_swap.rs

```diff
@@ -1204,6 +1209,7 @@ impl MakerSavedEvent {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct MakerSavedSwap {
     pub uuid: Uuid,
+    my_order_uuid: Option<Uuid>,
```

Comment:
Could you please tell, when the `my_order_uuid` is not equal to the `uuid` or is None?

Reply:
The UUID of the swap is set to taker order UUID as of now. So `my_order_uuid != uuid` for all maker swaps.

Reply:
I see, thanks!